projects/store/src/lib/store/interfaces/entity-adapter-reducers.ts

```diff
@@ -0,0 +1,19 @@
+import { EntityState } from '@ngrx/entity';
+
+// Iben: actions get typed to any as the type TypedAction<T> from ngrx/store is sadly not exported
+interface BasicEntityAdapterReducerActions {
```

Comment:
Is het de bedoeling dat deze niet exported is?

Reply:
Yup, enkel hetgeen wat naar buiten moet is geëxporteerd.